bcda/api/requests.go

```diff
@@ -245,7 +245,7 @@ func (h *Handler) JobStatus(w http.ResponseWriter, r *http.Request) {
 	switch job.Status {
 
 	case models.JobStatusFailed, models.JobStatusFailedExpired:
-		h.RespWriter.Exception(w, http.StatusInternalServerError, responseutils.InternalErr, "Service encountered numerous errors.  Unable to complete the request.")
+		h.RespWriter.Exception(w, http.StatusInternalServerError, responseutils.InternalErr, responseutils.DetailJobFailed)
```

Comment:
instead of responseutils.InternalErr being passed here, can we have new constant with the value of "Job Failed" (since this will be the detailsCode value of CreateOpOutcome in bcda/responseutils/writer.go https://github.com/CMSgov/bcda-app/blob/a862495f59a191e37fac1437ff2186243e33ce46/bcda/responseutils/writer.go#L130)